app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -32,7 +32,7 @@ import javax.inject.Inject
 typealias PatientUuid = UUID
 typealias FacilityUuid = UUID
 
-val DATE_OF_BIRTH_FORMAT_FOR_UI = DateTimeFormatter.ofPattern("d/MM/yyyy", Locale.ENGLISH)!!
+val DATE_OF_BIRTH_FORMAT_FOR_UI = DateTimeFormatter.ofPattern("dd/MM/yyyy", Locale.ENGLISH)!!
```

Comment:
Good point. This was being used to format only in the Search results for now. Should we make a new formatter for this screen? This wasn't a problem for Patient entry because we don't have a format a `LocalDate` there.